style.css

```diff
@@ -1,14 +1,130 @@
-div.text {
-  position: relative;
+body {
+    background: linear-gradient(
+            225deg,
+            rgba(195, 195, 195, 0.04) 0%,
+            rgba(195, 195, 195, 0.04) 25%,
+            rgba(234, 234, 234, 0.04) 25%,
+            rgba(234, 234, 234, 0.04) 50%,
+            rgba(107, 107, 107, 0.04) 50%,
+            rgba(107, 107, 107, 0.04) 75%,
+            rgba(132, 132, 132, 0.04) 75%,
+            rgba(132, 132, 132, 0.04) 100%
+        ),
+        linear-gradient(
+            65deg,
+            rgba(116, 116, 116, 0.04) 0%,
+            rgba(116, 116, 116, 0.04) 25%,
+            rgba(219, 219, 219, 0.04) 25%,
+            rgba(219, 219, 219, 0.04) 50%,
+            rgba(33, 33, 33, 0.04) 50%,
+            rgba(33, 33, 33, 0.04) 75%,
+            rgba(165, 165, 165, 0.04) 75%,
+            rgba(165, 165, 165, 0.04) 100%
+        ),
+        linear-gradient(
+            251deg,
+            rgba(38, 38, 38, 0.04) 0%,
+            rgba(38, 38, 38, 0.04) 25%,
+            rgba(223, 223, 223, 0.04) 25%,
+            rgba(223, 223, 223, 0.04) 50%,
+            rgba(35, 35, 35, 0.04) 50%,
+            rgba(35, 35, 35, 0.04) 75%,
+            rgba(203, 203, 203, 0.04) 75%,
+            rgba(203, 203, 203, 0.04) 100%
+        ),
+        linear-gradient(
+            236deg,
+            rgba(206, 206, 206, 0.04) 0%,
+            rgba(206, 206, 206, 0.04) 25%,
+            rgba(13, 13, 13, 0.04) 25%,
+            rgba(13, 13, 13, 0.04) 50%,
+            rgba(151, 151, 151, 0.04) 50%,
+            rgba(151, 151, 151, 0.04) 75%,
+            rgba(255, 255, 255, 0.04) 75%,
+            rgba(255, 255, 255, 0.04) 100%
+        ),
+        linear-gradient(
+            260deg,
+            rgba(133, 133, 133, 0.04) 0%,
+            rgba(133, 133, 133, 0.04) 25%,
+            rgba(169, 169, 169, 0.04) 25%,
+            rgba(169, 169, 169, 0.04) 50%,
+            rgba(91, 91, 91, 0.04) 50%,
+            rgba(91, 91, 91, 0.04) 75%,
+            rgba(74, 74, 74, 0.04) 75%,
+            rgba(74, 74, 74, 0.04) 100%
+        ),
+        linear-gradient(90deg, rgb(8, 35, 191), rgb(45, 136, 225));
 }
 
-div.text input {
-  padding-left: 15px;
+#container {
+    height: 550px;
+
```

Comment:
No need for empty lines between every CSS property. It increases your file size for no good reason. Grouping them together makes it easy to read and associate them together.